scripts/apps/vocabularies/views/vocabulary-config-modal.html

```diff
@@ -172,7 +172,7 @@ <h3 class="modal__heading" ng-show="!vocabulary._id"><span translate>Create</spa
 
         <div class="modal__footer">
             <button id="save-edit-btn" type="submit" class="btn btn--primary pull-right" ng-click="save()"
-                    ng-disabled="editForm.$invalid" translate>Save
+                    ng-disabled="!isDirty() || editForm.$invalid" translate>Save
```

Comment:
there is also `editForm.$dirty` that should be set on first change

Reply:
and also [editFrom.$pristine](https://docs.angularjs.org/api/ng/type/form.FormController#$pristine) which should be like !dirty